standard/expressions.md

```diff
@@ -1019,8 +1019,8 @@ For a function member that includes a parameter array, if the function member is
 - Otherwise, the expanded form is applicable if for each argument in `A`, one of the following is true:
   - the parameter-passing mode of the argument is identical to the parameter-passing mode of the corresponding parameter, and
     - for a fixed value parameter or a value parameter created by the expansion, an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter, or
-    - for an `in`, `out`, or `ref` parameter, the type of the argument expression is identical to the type of the corresponding parameter.
-  - the parameter-passing mode of the argument is value, and the parameter-passing mode of the corresponding parameter is input, and an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter.
+    - for a by-reference parameter, the type of the argument expression is identical to the type of the corresponding parameter.
+  - the parameter-passing mode of the argument is value, and the parameter-passing mode of the corresponding parameter is input, and an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter
```

Comment:
I think the last bullet should retain its full-stop. For the innermost list on 1020-22 I’d also introduce with a colon (“and:” on 1020) and seperate with a semi-colon (“parameter; or” on 1021). But will defer to @RexJaeschke on these.

Reply:
@Nigel-Ecma , those changes make sense to me; thank you!